test/number-is-even.test.js

```diff
@@ -0,0 +1,13 @@
+const { describe, it } = require('mocha');
+const assert = require('assert');
+const isEven = require('../src/number-is-even');
+
+describe('Verifica se o número é par', () => {
```

Comment:
```suggestion
describe('IsEven"', () => {
```

I think the function's number should be on the description